apps/frontend/src/locales/en/translations.json

```diff
@@ -666,7 +666,11 @@
         "reject": "Reject",
         "rejectText": "You voted to reject this proposal",
         "noVote": "No vote cast on this proposal",
-        "ineligibleVote": "Ineligible to vote - no SOV staked at proposal creation time <br/> Stake SOV to be eligible to vote on future proposals"
+        "ineligibleVote": "Ineligible to vote - no SOV staked at proposal creation time <br/> Stake SOV to be eligible to vote on future proposals",
```

Comment:
You shouldn't have HTML tags in translations - break it into multiple translations instead and style it in the component.

Reply:
I didn't add it, but I will fix it